### PR TITLE
Feature/14.08.2019_TNT794_Register_MyIssues_Maksymiv

### PR DESCRIPTION
**Pull request by VitaliiMaksymiv**

### CODE REVIEW

**Review by andriyshevchenko — DISMISSED** [review]
Molodec

File: TransIT.API/Controllers/IssueController.cs [hunk 1]
```diff
@@ -188,5 +189,27 @@ private async Task<ulong> GetTotalRecordsForCurrentUser(bool isCustomer)
...
+            ulong totalAmount,
+            string errorMessage = "")
+        {
+            return new DataTableResponseDTO
+            {
+                Draw = (ulong)model.Draw,
+                Data = res.ToArray(),
+                RecordsTotal = totalAmount,
+                RecordsFiltered =
+                    (model.Filters?.Count > 1
```
Comment (igolovchuk) [inline TransIT.API/Controllers/IssueController.cs]: .Any() == Count > 0, so you can remove .Any() check

File: TransIT.API/Controllers/IssueController.cs [hunk 2]
```diff
@@ -188,5 +189,27 @@ private async Task<ulong> GetTotalRecordsForCurrentUser(bool isCustomer)
...
+                     && model.Filters.Any())
+                    || (model.Search != null
```
Comment (igolovchuk) [inline TransIT.API/Controllers/IssueController.cs]: replace with just !string.IsNullOrEmty(model.Search?.Value)

**Review by BohdanBuhrii — DISMISSED** [review]

**Review by andriyshevchenko — APPROVED** [review]

**Review by andriyshevchenko — DISMISSED** [review]

**Review by SviatoslavBachynskyi — DISMISSED** [review]

File: TransIT.API/Controllers/IssueController.cs [hunk 3]
```diff
@@ -188,5 +189,25 @@ private async Task<ulong> GetTotalRecordsForCurrentUser(bool isCustomer)
...
+            string errorMessage = "")
+        {
+            return new DataTableResponseDTO
+            {
+                Draw = (ulong)model.Draw,
+                Data = res.ToArray(),
+                RecordsTotal = totalAmount,
+                RecordsFiltered =
+                    (model.Filters?.Count > 1)
+                    || (!string.IsNullOrEmpty(model.Search.Value))
```
Comment (igolovchuk) [inline TransIT.API/Controllers/IssueController.cs]: model.Search** ?. **Value

**Review by igolovchuk — CHANGES_REQUESTED** [review]
you missed null check

**Review by igolovchuk — APPROVED** [review]

**Review by andriyshevchenko — APPROVED** [review]

**Review by BohdanBuhrii — APPROVED** [review]

**Pull request merged**